### PR TITLE
Add support of Symfony 3.0

### PR DESCRIPTION
**Pull request by francoispluchino**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #84
| License       | MIT
| Doc PR        | ~

### CODE REVIEW

File: src/Provider/FileProvider.php [hunk 1]
```diff
@@ -16,7 +16,7 @@ class FileProvider extends AbstractProvider
 
     public function __construct($name, array $options, $client, Cache $cache, Logger $logger) {
         $this->name     = $name;
-        /** @var md5 only contain numeric and A to F, so it is file system safe */
+        /** md5 only contain numeric and A to F, so it is file system safe */
```
Comment (xabbuh) [inline src/Provider/FileProvider.php]: The comment should not start with `/**` which indicates a docblock.

**Comment by xabbuh** [comment]
There is no job on Travis performing a build with Symfony 3.

**Comment by francoispluchino** [comment]
It's done.

**Comment by francoispluchino** [comment]
Travis test Symfony 3.0 with PHP 5.4. What is the config for testing Symfony 3.0 with PHP 5.5 and greater?

**Comment by francoispluchino** [comment]
The config of Travis is fixed now.

**Comment by xabbuh** [comment]
:+1: LGTM

The Travis CI configuration can be improved but that should better be done in a separate pull request.

**Comment by francoispluchino** [comment]
Ok, I'm interested by your optimization of the Travis config.

**Comment by xabbuh** [comment]
@francoispluchino You can take a look at my changes in #86 (one job is currently failing because of a YAML syntax error that will be fixed by #87).

**Comment by francoispluchino** [comment]
@xabbuh Thanks!

File: composer.json [hunk 2]
```diff
@@ -19,18 +19,18 @@
     "require": {
         "php":                      ">=5.4.0",
         "doctrine/common":          "~2.4",
-        "symfony/dependency-injection": "~2.3, <3.0.0@dev",
-        "symfony/config": "~2.3, <3.0.0@dev",
-        "symfony/http-kernel": "~2.3, <3.0.0@dev",
-        "symfony/console": "~2.3, <3.0.0@dev",
-        "symfony/monolog-bundle":   "~2.3, <3.0.0@dev"
+        "symfony/dependency-injection": "~2.3|~3.0",
```
Comment (aequasi) [inline composer.json]: while you are in here, change the `~`'s to `^`'s

File: src/Provider/IronMqProvider.php [hunk 3]
```diff
@@ -46,7 +46,7 @@ class IronMqProvider extends AbstractProvider
     /**
      * IronMQ Queue
      *
-     * @var stdObject
+     * @var object
```
Comment (aequasi) [inline src/Provider/IronMqProvider.php]: object isn't a valid type in PHP, \stdClass may make more sense

File: composer.json (hunk 2 above)
Comment (francoispluchino) [inline composer.json]: `^3.0.0`?

File: src/Provider/IronMqProvider.php (hunk 3 above)
Comment (francoispluchino) [inline src/Provider/IronMqProvider.php]: `stdObject` doesn't exist, only `stdClass`. `object` is a typeint (see [PHP doc object](http://php.net/manual/en/language.types.object.php)).

Comment (aequasi) [inline src/Provider/IronMqProvider.php]: Ah, alright, nvm then

File: composer.json (hunk 2 above)
Comment (aequasi) [inline composer.json]: i would actually say `^3.0`, but you could change the other `~`'s too

Comment (francoispluchino) [inline composer.json]: Ok I do this.

**Comment by francoispluchino** [comment]
It's ok for you now?

**Comment by aequasi** [comment]
@kmfk

**Comment by kmfk** [comment]
@francoispluchino Thanks!

**Pull request merged**